modules/tensor_mechanics/doc/content/source/vectorpostprocessors/AverageSectionValueSampler.md

```diff
@@ -0,0 +1,45 @@
+# AverageSectionValueSampler
+
+!syntax description /VectorPostprocessors/AverageSectionValueSampler
+
+## Description
+
+This vector postprocessor locates a number of nodes in a structural component and computes their average
+variables along the $X$, $Y$, and $Z$ axes on multiple cross sections. This feature can be used to compute,
+for example, average displacements.
```

Comment:
```suggestion
This computes the average value of specified nodal variables at nodes located within cross-sectional planes
at user-specified locations along the axis of a mesh.  This is designed to operate on bodies whose meshes are created by extrusion, so that sets of nodes that make up the mesh lie within planes.
```